src/AppBundle/Document/DemoSeoExtractor.php

```diff
@@ -24,11 +24,7 @@
  *
  * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
  */
-class DemoSeoExtractor extends DemoSeoContent implements
-    TitleReadInterface,
-    DescriptionReadInterface,
-    OriginalUrlReadInterface,
-    KeywordsReadInterface
+class DemoSeoExtractor extends DemoSeoContent implements TitleReadInterface, DescriptionReadInterface, OriginalUrlReadInterface, KeywordsReadInterface
```

Comment:
oh. we should use that everywhere, cause i saw that on seo-bundle yesterday too. from which coding style is that?